libs/zedUpload/azureutil/azure.go

```diff
@@ -24,7 +24,7 @@ import (
 const (
 	// SingleMB contains chunk size
 	SingleMB       int64 = 1024 * 1024
-	blobURLPattern       = "https://%s.blob.core.windows.net/%s"
+	blobURLPattern       = "https://%s.blob.core.windows.net"
```

Comment:
This URL string makes it look like one can contol a prefix which doesn't make sense if the user wants to control the whole host part of the URL.

Reply:
Ah - that is the accountname position. My confusion.

Reply:
I leave it here to save compatibility with empty fqdn.